README.md

```diff
@@ -172,6 +172,24 @@ Additionally, required for tests:
 5) Required packages:
    - liblevel-zero-dev (Linux) or level-zero-sdk (Windows)
 
+#### DevDax memory provider (Linux only)
```

Comment:
pls add also a web docs (in `scripts/docs_config/api.rst`)

Reply:
Done